extensions/sns/e2e/tests/sns.bash

```diff
@@ -88,9 +65,10 @@ SNS_CONFIG_FILE_V2_NAME="sns_v2.yml"
     cat dfx.json
     # Deploy the SNS
     install_asset sns/valid
-    dfx sns config validate
+    dfx sns init-config-file validate
     # The remaining steps don't work any more as a pre-launch whitelist has been added.
-    #dfx sns deploy
+    # and the steps required have changed due to one-proposal
+    #dfx sns propose
```

Comment:
As is the steps will not work because we need to supply a neuron ID to vote in the proposal when calling `dfx sns propose` and I'm not sure what ID to use

Reply:
But I removed the "a pre-launch whitelist has been added" comment!